README.md

```diff
@@ -30,6 +30,19 @@ or in single-file mode.
 5. (WIP) Runs some custom fixers based on `flake8-bugbear`
 6. Iterates those steps until the source code stops changing.
 
+## Using with pre-commit
+
+If you use [pre-commit](https://pre-commit.com/), you can use it with Shed by
+adding the following to your `.pre-commit-config.yaml`:
+
+```yaml
+repos:
+- repo: https://github.com/Zac-HD/shed.git
```

Comment:
No need for `.git`:
```suggestion
- repo: https://github.com/Zac-HD/shed
```

Compare https://github.com/asottile/pyupgrade#as-a-pre-commit-hook